spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/data/TestSparkParquetReader.java

```diff
@@ -132,13 +145,14 @@ protected List<InternalRow> rowsFromFile(InputFile inputFile, Schema schema) thr
     }
   }
 
-  protected Table tableFromInputFile(InputFile inputFile, Schema schema) throws IOException {
+  protected Table tableFromInputFile(InputFile inputFile, Schema schema, int formatVersion)
```

Comment:
It looks like this change isn't needed since this is only called from the INT96 test. Tests pass if I revert these changes, so I think this must be from an older version of the variant test?